### PR TITLE
Versions

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

File: caching-service/pom.xml [hunk 1]
```diff
@@ -5,16 +5,21 @@
 
     <groupId>org.jboss.quickstarts.jdg</groupId>
     <artifactId>caching-service</artifactId>
-    <version>7.2.2.redhat-SNAPSHOT</version>
+    <version>7.3.0-SNAPSHOT</version>
 
     <name>JBoss JDG Quickstart: Caching Service</name>
 
+    <parent>
```
Comment (galderz) [inline caching-service/pom.xml]: Why is this needed?

Even if it's needed, it should be pointing to the JDG version of the parent.

**Review by galderz — CHANGES_REQUESTED** [review]

File: caching-service/pom.xml (hunk 1 above)
Comment (karesti) [reply caching-service/pom.xml]: I added the parent as I was told in a private conversation, and at first nothing was building and the pom parent of the quickstarts is not the parent of any of them. 
I've removed everywhere, and indeed, it seems not necessary so I'm removing them all. The bom's parent is the infinispan-build-configuration-parent in datagrid.

Comment (karesti) [reply caching-service/pom.xml]: well, if i don't put it in the hadoop example, it fails. Now I'm thinking that I have all the dependencies locally, so I'm rebuilding the examples from an empty maven repository.

Comment (karesti) [reply caching-service/pom.xml]: @galderz updated

**Comment by karesti** [comment]
@galderz is it fine for you now Galder ?

**Comment by galderz** [comment]
Looks good. Is there a CI run for this?

**Comment by galderz** [comment]
@karesti Running locally I see:

```
[ERROR] The build could not read 4 projects -> [Help 1]
[ERROR]
[ERROR]   The project org.jboss.quickstarts.jdg:jboss-spring-jdg:7.3.0-SNAPSHOT (/Users/g/1/jboss-jdg-quickstarts/spring/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.jboss.spec.javax.transaction:jboss-transaction-api_1.1_spec:jar is missing. @ line 70, column 21
[ERROR]
[ERROR]   The project org.jboss.quickstarts.jdg:jboss-spring-boot-session-tomcat:7.3.0-SNAPSHOT (/Users/g/1/jboss-jdg-quickstarts/spring-session/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.jboss.spec.javax.transaction:jboss-transaction-api_1.1_spec:jar is missing. @ line 65, column 21
[ERROR]
[ERROR]   The project org.jboss.quickstarts.jdg:jboss-carmart-tx:7.3.0-SNAPSHOT (/Users/g/1/jboss-jdg-quickstarts/carmart-tx/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.jboss.spec.javax.transaction:jboss-transaction-api_1.1_spec:jar is missing. @ line 180, column 21
[ERROR]
[ERROR]   The project org.jboss.quickstarts.jdg:jboss-hadoop-quickstart:7.3.0-SNAPSHOT (/Users/g/1/jboss-jdg-quickstarts/hadoop/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.apache.flink:flink-java:jar is missing. @ line 80, column 21
```

With this from root:

```bash
mvn -DskipTests=true clean install -s settings.xml
```

**Comment by galderz** [comment]
@karesti Is this fixed?

**Comment by karesti** [comment]
@galderz still need to do by hand the mvn clean install for ``` jdg/build-configuration```

**Comment by galderz** [comment]
@karesti Merging this for now, please sort out CI as soon as possible.

**Pull request merged**